### DISCUSSION TITLE
Is there a way to add the field name being validated

### DISCUSSION THREAD

**Discussion by HosMercury** [maxcountryman/axum-messages#7] [Q&A] [answered]
like 
```
Level: Error
field: username
message: "username is already exists
```

for a reason: to color that field in HTML with red color around?
where or how could I add a field name?

**Comment by maxcountryman** [comment, marked as answer]
You're probably looking for something like [extra tags](https://docs.djangoproject.com/en/4.2/ref/contrib/messages/#adding-extra-message-tags), which Django's implementation provides. This isn't something we've currently implemented (we don't have a concept of "tags" at all as it stands).
